src/cider/nrepl/middleware/reload.clj

```diff
@@ -37,7 +37,7 @@
         reload (user-reload 'reload reload/reload)
         unload (user-reload 'unload reload/unload)]
     (cond
-      (:all msg)   (reload (assoc opts :all true))
+      (:all msg)   (reload (assoc opts :only :all))
```

Comment:
This is the correct option to reload everything: https://github.com/tonsky/clj-reload?tab=readme-ov-file#usage-choose-what-to-reload